rules/windows/execution_suspicious_image_load_wmi_ms_office.toml

```diff
@@ -10,6 +10,7 @@ Identifies a suspicious image load (wmiutils.dll) from Microsoft Office processe
 adversarial activity where child processes are spawned via Windows Management Instrumentation (WMI). This technique can
 be used to execute code and evade traditional parent/child processes spawned from MS Office products.
 """
+from = "now-9m"
```

Comment:
i thought we weren't doing both `timestamp_override` and `from`

Reply:
https://github.com/elastic/detection-rules/pull/945#discussion_r578020626

as long as we are consistent either way - so either strip them all or add to the ones that were missing